modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/shared/components/filter/util/filterUtil.es.js

```diff
@@ -73,10 +73,28 @@ const getSelectedItems = filterResults => {
 			? filter.items.filter(item => item.active)
 			: [];
 
-		return !!filter.items.length;
+		return filter.items.length > 0;
 	});
 };
 
+const handleFilterItems = (items, selectedKeys) =>
+	items.map((item, index) => {
+		const key = item.key || String(item.id);
+
+		return {
+			...item,
+			active: selectedKeys && selectedKeys.includes(key),
+			dividerAfter: item.dividerAfter && !!items[index + 1],
+			key
+		};
+	});
+
+const mergeItemsArray = (items1, items2) => {
+	const mergedItems = items1 || [];
+
+	return items2 ? mergedItems.concat(items2) : mergedItems;
+};
```

Comment:
Can you replace it to:
```javascript
const mergeItemsArray = (items = [], newItems = []) => {
    return items.concat(newItems);
};
```
?

Reply:
I replaced to another form and made some improvements
`const mergeItemsArray = (baseItems = [], ...items) => {
	items = items.filter(value => value !== undefined && value !== null);
	return baseItems.concat(...items);
};`